packages/web-components/README.md

```diff
@@ -49,8 +49,7 @@ For quick start, you can use our pre-built bundle that contains masthead, footer
 <html>
   <head>
     <script type="module">
-      // Copy from `dist` directory in the package and put it to the same directory as this file
-      import './ibmdotcom-web-components-dotcom-shell.min.js';
+      import 'https://www.ibm.com/common/carbon-for-ibm-dotcom/v0.5.0/ibmdotcom-web-components-dotcom-shell.min.js';
```

Comment:
@asudoh you can use:
`https://www.ibm.com/common/carbon-for-ibm-dotcom/latest/ibmdotcom-web-components-dotcom-shell.min.js`

Reply:
Thanks @jeffchew! Updated.